cypress/page/address-step.page.ts

```diff
@@ -2,7 +2,9 @@ class AdressStepPage {
     private proceedToCheckoutAdress: string;         
 
     constructor() {        
-        this.proceedToCheckoutAdress = ".cart_navigation span";
+        this.proceedToCheckoutAdress = "[type='submit'][name='processAddress']";
```

Comment:
you can add the tag type instead of the submit to not use two attributes: `button[name='processAddress']`

Reply:
=O 
Now I understand something that I didn´t understand about using tags (I think so)
Thank you!

Nevertheless, I though that using two attributes was a very good selector, wasn´t it?

Reply:
well in this case, is more readable to use tag + attribute than use two attributes. I do not have more reasons, I don't know which of the two options is more efficient.